src/util/numeric.cpp

```diff
@@ -8,6 +8,13 @@
 
 namespace stellar
 {
+
+int64_t
+roundedDivide(int64_t A, int64_t B, Rounding rounding)
```

Comment:
This function isn't safe. First, it doesn't protect against overflow or division-by-zero. Second, it isn't correct for negative `B` because `B < 0` implies
```
(A + B - 1) / B <= (A - 1) / B <= A/B
```
so it rounds down when it should round up in this case. As an explicit example, suppose `A = 1` and `B = -1`. Then
```
(A + B - 1) / B = (1 + (-1) - 1) / (-1) = (-1) / (-1) = 1
```
doesn't even have the correct sign. Given all these issues, let's just remove this function and use an appropriate `bigDivide` function instead.

Reply:
yeah I was using the `bigDivide` function before and Rafal made the observation that it's not needed (so we don't need to deal with overflow/negative numbers).
I am reverting the change: it's not worth having this function that is too limited in the code base